client-rpc/src/rpc/staking_rpc.rs

```diff
@@ -463,6 +462,8 @@ fn get_node_metadata(validator_name: &str, validator_pubkey: &str) -> Result<Cou
         security_contact: None,
         consensus_pubkey: TendermintValidatorPubKey::Ed25519(pubkey_bytes),
         // FIXME real keypackage
-        confidential_init: mock_confidential_init(),
+        confidential_init: ConfidentialInit {
```

Comment:
this is needed for the integration tests?

Reply:
yes, because the reference time in integration test can't be controlled, so it'll need real keypackage or other mock solutions.